bukkit/src/main/java/hu/montlikadani/tablist/bukkit/tablist/groups/GroupPlayer.java

```diff
@@ -38,21 +42,15 @@ public ITabScoreboard getTabTeam() {
 
 	public void setGroup(TeamHandler group) {
 		this.group = group;
+		this.groups.setToSort(true);
 	}
 
 	public TeamHandler getGroup() {
 		return group;
 	}
 
 	public String getFullGroupTeamName() {
-		String name = Integer.toString(100000 + getPriority())
-				+ (group == null ? tabListUser.getPlayer().getName() : group.getTeam());
-
-		if (name.length() > 16) {
-			name = name.substring(0, 16);
-		}
-
-		return name;
+		return "tablist" + safePriority;
```

Comment:
Scoreboard team names must not be larger than 16 characters even when we're using packets for creating teams.